src/Files.App/Views/Properties/HashesPage.xaml

```diff
@@ -79,14 +65,43 @@
 							Grid.Column="2"
 							VerticalAlignment="Center"
 							Style="{StaticResource BodyTextBlockStyle}"
-							Text="Hash value" />
+							Text="{helpers:ResourceString Name=HashValue}" />
+						<Button
+							x:Name="SelectAlgorithmsButton"
+							Grid.Column="3"
+							Padding="6"
+							AutomationProperties.Name="{helpers:ResourceString Name=SelectHashesToShow}"
+							Background="Transparent"
+							BorderBrush="Transparent"
+							ToolTipService.ToolTip="{helpers:ResourceString Name=SelectHashesToShow}">
+							<FontIcon FontSize="14" Glyph="&#xE7B3;" />
+							<Button.Flyout>
+								<MenuFlyout x:Name="SelectAlgorithmsMenuFlyout" Closing="MenuFlyout_Closing">
+									<ToggleMenuFlyoutItem Text="{x:Bind HashesViewModel.Hashes[0].Algorithm}"
+														  IsChecked="{x:Bind HashesViewModel.Hashes[0].IsEnabled, Mode=TwoWay}"
+														  Click="ToggleMenuFlyoutItem_Click" />
+									<ToggleMenuFlyoutItem Text="{x:Bind HashesViewModel.Hashes[1].Algorithm}"
+														  IsChecked="{x:Bind HashesViewModel.Hashes[1].IsEnabled, Mode=TwoWay}"
+														  Click="ToggleMenuFlyoutItem_Click" />
+									<ToggleMenuFlyoutItem Text="{x:Bind HashesViewModel.Hashes[2].Algorithm}"
+														  IsChecked="{x:Bind HashesViewModel.Hashes[2].IsEnabled, Mode=TwoWay}"
+														  Click="ToggleMenuFlyoutItem_Click" />
+									<ToggleMenuFlyoutItem Text="{x:Bind HashesViewModel.Hashes[3].Algorithm}"
+														  IsChecked="{x:Bind HashesViewModel.Hashes[3].IsEnabled, Mode=TwoWay}"
+														  Click="ToggleMenuFlyoutItem_Click" />
+									<ToggleMenuFlyoutItem Text="{x:Bind HashesViewModel.Hashes[4].Algorithm}"
+														  IsChecked="{x:Bind HashesViewModel.Hashes[4].IsEnabled, Mode=TwoWay}"
+														  Click="ToggleMenuFlyoutItem_Click" />
+								</MenuFlyout>
+							</Button.Flyout>
+						</Button>
 					</Grid>
 				</ListView.Header>
 
 				<!--  Template  -->
 				<ListView.ItemTemplate>
 					<DataTemplate x:DataType="models:HashInfoItem">
-						<ListViewItem IsSelected="{x:Bind IsSelected, Mode=TwoWay}">
+						<ListViewItem IsSelected="{x:Bind IsSelected, Mode=TwoWay}" Visibility="{x:Bind IsEnabled, Mode=OneWay}">
 							<Grid ColumnSpacing="12">
```

Comment:
- Use Converter 'BoolToVisibility' since IsEnabled must be boolean.
- Remove Mode=OneWay due to a warning WMC1506.

Reply:
> Use Converter 'BoolToVisibility' since IsEnabled must be boolean.

I thought so too, but it actually works as it is.

> Remove Mode=OneWay due to a warning WMC1506.

No. The source property change is not detected without `Mode=OneWay`